site/content/en/docs/technical-outcomes/scheduling/technical-outcomes/simulation/scenario-4/README.demo

```diff
@@ -0,0 +1,40 @@
+# Scheduling pods with a limit Range
+
+## Create cluster
+kwokctl create cluster
+
+## View clusters
+kwokctl get clusters
+
+## Deploy node
+kubectl apply -f node.yaml
+
+## Wait for a while so the node is running
+sleep 5
+
+## Create a resource limit
+kubectl apply -f limit-range.yaml
+
+## Confirm the limit has the required values
+kubectl describe limitranges cpu-resource-constraint
+
+## Wait for a while so you can observe the result
+sleep 10
+
+## Deploy a pod above the resource limit
+kubectl apply -f pod-beyond-limit.yaml
```

Comment:
The error output from this command makes the asciinema recorder quit suddenly. We may need to find a way around it or skip adding an `svg` image in this scenario for now.